### ISSUE TITLE
Question about the install prerequisites

### ISSUE THREAD

**Issue by JackieMium** [i-rinat/freshplayerplugin#303]
I am new to Linux and this is maybe a stupid question. I am using Gentoo 64bit.
I got it all working now after installing some of the listed install prerequisites. Now I am curious as I HAVE got it working fine. Are those install prerequisites still needed or is it that they are just necessary to compile phase? Aka, are those  install prerequisites safe to remove?
Sorry if this is the wrong place to ask or this is a so stupid question.

**Comment by i-rinat** [comment]
Well, cmake, gcc, g++, pkg-config, and ragel are used only during compilation phase. They can be removed after without effect. Others are development packages, headers and unversioned library symlinks. They also can be removed, if you keep actual libraries installed. Perhaps, Gentoo doesn't split libraries into multiple packages. In that case you should keep them.

**Comment by JackieMium** [comment]
Okay, got it. 
Thanks for your help.

**Comment by bootkiller** [comment]
@JackieMium Don't _ever_ remove those packages like that, always use your distributions package manager to do it for you. https://wiki.gentoo.org/wiki/Knowledge_Base:Remove_orphaned_packages

**Comment by JackieMium** [comment]
@bootkiller Yes, Gentoo uses Portage to manager PKGs and the way you mentioned is the proper one to  remove those PKGs. However, the 6 PKGs I mentioned above were installed using `emerge --oneshot PKG`and the `--oneshot` option is to avoid those PKGs from adding to the `world` list. Because adding everything you need into the `world` list is not the good way for Portage to manage the system and all its PKGs, Portage will have to take a long time to calculate dependencies by doing that. The best way is to add _freshplayerplugin_ (in this example) in the world list and this will automatically pulls in all its dependencies(ie the 6 PKGs here). However, the problem here is that you cannot add _freshplayerplugin_ to your world list.

**Comment by grray** [comment]
@JackieMium I did today
`emerge -av freshplayerplugin`
and got it working in firefox. Without any other dependencies

**Comment by JackieMium** [comment]
@grray My mistake. I did search in Portage by `eix -S fresh` and didn't see `freshplayerplugin`. Today I searched via `eix -s fresh` and there `freshplayerplugin` was. Suppose it's in Gentoo Portage and I just didn't know. Thanks all guys for your patience.
PS: @i-rinat It occurred to me that it might be a good idea to put something like `Gentoo users can get freshpayerplugin in official Portage` in README.
